lib/levels.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+var flatstr = require('flatstr')
+
+var levels = {
+  fatal: 60,
+  error: 50,
+  warn: 40,
+  info: 30,
+  debug: 20,
+  trace: 10
+}
+module.exports.levels = levels
```

Comment:
What do you think about moving the whole exports to the bottom of the file.
I usually just use `module.exports`
```
module.exports = {
  nums: nums,
  lscache: lscache,
  isStandardLevel: isStandardLevel,
  isStandardLevelVal: isStandardLevelVal
}
```

That way you can make sure that there's nothing else exported.
IMO it improves the readability a bit.